docs/build-on-morph/developer-resources/node-operation/upgrade-node/0-centralized-sequencer-upgrade.md

```
@@ -0,0 +1,60 @@
+
+   ```bash
+   make stop-node && make run-node          # mainnet (Docker)
+   make stop-node && make run-hoodi-node    # Hoodi (Docker)
+   ```
+5. **Confirm it is following the chain** — see [Verify](#verify) below.
+
+## Verify
+
+Check sync status as usual (see [Run a full node → Verify the Node](../full-node/1-run-in-docker.md#verify-the-node)). Every node verifies batches against L1; if it detects a mismatch you will see a log line such as:
+
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)